simple/stats/config.md

```diff
@@ -36,7 +36,16 @@ The config parameters for the files to be imported should be specified in a `con
         "Provenance2 Name": "http://source1.com/provenance2"
       }
     }
-  }
+  },
+  "database": {
+    "type": "cloudMySql",
+    "params": {
+        "instance": "datcom-website-dev:us-central1:dc-graph",
+        "user": "root",
+        "password": "secret",
+        "db": "datacommons"
```

Comment:
the user and password should be part of the `docker run` command line arg, rather than in the config file, which is usually checked in?

Reply:
Ack. Let me do this in a separate PR.